src/components/shared/banners-as-footer-mobile/EsBannersAsFooterMobile.vue

```diff
@@ -0,0 +1,55 @@
+<!-- eslint-disable -->
+<template>
+    <div class="banners-mobile d-lg-none d-md-none d-sm-none">
+      <img v-for="(image, index) in images" :src="image.src" :alt="image.alt" :key="index" />
+    </div>
+</template>
+
+<script lang="ts">
+  import Vue from 'vue';
+
+  export default Vue.extend({
+    name: 'es-banners-as-footer-mobile',
+
+    props: {
+      fitness: {
```

Comment:
let's add a default value in here, in case something happens and the prop is not sent